tests/testthat/test_subset_datarqst.R

```diff
@@ -0,0 +1,57 @@
+context("Test that subset_datarqst() functions as expected...")
+
+test_PopIDs <- c('CHO', 'BAN', 'VEL', 'SSQ', 'HOC', 'PIL', 'EDM')
+test_data <- run_pipelines(PopID = test_PopIDs, save = FALSE)
```

Comment:
We have a `pipeline_output` object that is created when we build the tests. You should use this for the tests rather than calling `run_pipelines` again, just so the tests don't take too long.

Reply:
Thanks for the tip, I was not aware that we'll have the `pipeline_output`available for this. Have adjusted both the function and the tests accordingly.